testid_test.go

```diff
@@ -5,19 +5,21 @@ import (
 	"testing"
 
 	"github.com/google/uuid"
+	"github.com/newmo-oss/gotestingmock"
```

Comment:
By introducing this new module, we should execute `go mod tidy` to update `go.mod` file.